src/teacher.py

```diff
@@ -192,3 +192,33 @@ def get_responses(self) -> List[Optional[Response]]:
     def get_query_cost(self) -> float:
         """Return the query cost."""
         return self._query_cost
+
+
+@dataclass
+class TeacherInteractionMonitorWithVideo(TeacherInteractionMonitor,
+                                         utils.VideoMonitor):
+    """A monitor that wraps interaction with an environment to include
+    generating and answering queries.
+
+    Optionally, if CFG.make_interaction_videos is True, also renders
+    every state and action encountered. The render_fn is generally
+    env.render.
+    """
+    _request: InteractionRequest
```

Comment:
we don't need these 4 lines right? cuz they're automatically handled by the inheritance?